src/globalStyles.css

```diff
@@ -0,0 +1,4 @@
+* {
```

Comment:
I would recommend to use the `createGlobalStyles` function by styled-components instead of a CSS file.